Resources/Maps/_Goobstation/barratry.yml

```
@@ -78103,6 +78312,39 @@ entities:
+      parent: 13815
+    - type: Paper
+      stampState: paper_stamp-hos
+      stampedBy:
+      - stampedColor: '#CC0000FF'
+        stampedName: stamp-component-stamped-name-hos
+      content: >-
+        [color=#cb0000]░█▄▄█░ [head=2]Security Memo[/head]
+
+        ██▄▄██
+
+        ▀█▄▄█▀ 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)